js/npm/api-keys/createKey.js

```diff
@@ -17,7 +17,7 @@ async function createApiKeySample() {
         featureFlagOne: true,
         featureFlagTwo: false,
       },
-      expiry: new Date("2023-09-01 00:00:00"), // optional expiry date
```

Comment:
While this works fine, `expiry` is expected to be a `Date` object, we want the users to use the library just like any other JS code, passing `expiry` as a date feels more natural in a JS environment

Reply:
See the comment before. Can you test using the JS Date object?